docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG TAG=11.0
+ARG MDB_TAG=$TAG
+FROM mariadb:${MDB_TAG}
```

Comment:
usually we start all Dockerfiles with `FROM` and `MAINTAINER`, then add additional stuff. I suggest to keep it consistent and move `ARG` below these lines.

Reply:
We actually need to start with the MDB_ARG, because we need it for referring to the base mariadb image to start from. It comes as an build option. But I've got rid of the additional `TAG` argument, though.